### PR TITLE
add peerNum to store.global(blocked)

### PR DESCRIPTION
**Pull request by amazingandyyy**

### CODE REVIEW

**Comment by amazingandyyy** [comment]
wait for #741.

File: src/components/BottomBar/index.js [hunk 1]
```diff
@@ -60,13 +46,13 @@ export default class BottomBar extends Component {
 const BlockInfo = withStyles(styles)(({ classes, blockNum, blockTime }) => (
   <Grid item xs={12} md={6} className={classes.bottomBarBlockInfoWrapper}>
     <Typography variant="body1">
-      <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />: {blockNum}</span>
-      <FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />: {blockTime ? getShortLocalDateTimeString(blockTime) : ''}
+      <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />:&nbsp;{blockNum}</span>
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: can we do this as a string template? `: ${blockNum}`

File: src/components/BottomBar/index.js [hunk 2]
```diff
@@ -60,13 +46,13 @@ export default class BottomBar extends Component {
...
+      <FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />:&nbsp;{blockTime ? getShortLocalDateTimeString(blockTime) : ''}
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: string template

File: src/components/BottomBar/index.js [hunk 3]
```diff
@@ -76,7 +62,11 @@ const NetworkConnection = withStyles(styles)(({ classes }) => (
       )}
       <span>
         {navigator.onLine ? (
-          <FormattedMessage id="bottomBar.online" defaultMessage="Online" />
+          <Fragment>
+            <FormattedMessage id="bottomBar.online" defaultMessage="Online" />
+            :&nbsp;{peerNum}&nbsp;
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: string template

File: src/components/BottomBar/index.js (hunk 2 above)
Comment (amazingandyyy) [reply src/components/BottomBar/index.js]: okok

File: src/components/BottomBar/index.js (hunk 1 above)
Comment (amazingandyyy) [reply src/components/BottomBar/index.js]: good idea

**Comment by amazingandyyy** [comment]
waiting for https://github.com/bodhiproject/bodhi-server/pull/58/files

**Pull request merged**